hi-ml-histopathology/src/histopathology/models/deepmil.py

```diff
@@ -113,7 +74,7 @@ def get_classifier(self) -> Callable:
         else:
             raise ValueError(f"Dropout rate should be in [0, 1), got {self.dropout_rate}")
 
-    def get_loss(self) -> Callable:
+    def get_loss(self) -> Callable:  # TODO: MAKE LIST
```

Comment:
here and at the other places where you have the "MAKE LIST" comment: Are you planning to fix that in this PR?